frameworks/helloworld/tests/test_taskcfg.py

```diff
@@ -29,7 +30,8 @@ def configure_package(configure_security):
 
 @pytest.mark.sanity
 def test_deploy():
-    wait_time = 30
+    wait_time = 600
+    sdk_plan.wait_for_kicked_off_deployment(config.SERVICE_NAME)
```

Comment:
As @porridge points out in https://jira.mesosphere.com/browse/DCOS-40639 this may not be enough to ensure scheduler health, but that is an orthogonal problem.

Reply:
Yeah, for this use case wait_for_kicked_off_deployment should be fine. Would be good to revisit for a future PR.